playbooks/tests/tasks/integration.yml

```diff
@@ -25,6 +25,10 @@
            external | awk '/ external / {print $2}' ); nova add-floating-ip
            turtle-stack ${FLOATING_IP}; sleep 60
 
+  - name: get floating IP
+    shell: . /root/stackrc; nova show turtle-stack | grep 'internal network' | awk '{ print $6 }'
```

Comment:
I updated my PR to be your commit with the change to $5 so that we could get this in tonight and all the other pending 1.2.x stuffs.